linalg/sparsemat.cpp

```diff
@@ -3718,9 +3718,9 @@ SparseMatrix *RAP(const SparseMatrix &Rt, const SparseMatrix &A,
    SparseMatrix * R = Transpose(Rt);
    SparseMatrix * RA = Mult(*R,A);
    delete R;
-   SparseMatrix * out = Mult(*RA, P);
+   SparseMatrix * mout = Mult(*RA, P);
    delete RA;
-   return out;
+   return mout;
```

Comment:
Personally I am not a fan of the prefix + `out` pattern. Will the reader understand that `mout` stands for "matrix output" even if all traces of `out` are gone?

Other options could be `rap`, `RAP_matrix`, `result`, `output`, etc.

Reply:
The previous function uses `RAP_` so I suggest we reuse that.